conda/lock-environment.sh

```diff
@@ -36,6 +36,9 @@ sed -i.bak -r 's/--hash=md5:None//' conda-linux-64.lock.yml
 sed -i.bak -r 's/--hash=md5:None//' conda-osx-64.lock.yml
 rm *.yml.bak
 
+# Temporary fix for https://github.com/conda-incubator/conda-lock/issues/179
+echo '    - snowexsql === 0.2.0' >> conda-linux-64.lock.yml
+echo '    - snowexsql === 0.2.0' >> conda-osx-64.lock.yml
```

Comment:
not ideal hack (`conda-lock install` will omit snowexsql), but this will at least get it into the docker image and work for people following our installation and contributing instructions